packages/tldraw/src/lib/tools/SelectTool/DragAndDropManager.ts

```diff
@@ -137,8 +137,8 @@ export class DragAndDropManager {
 		this.first = true
 	}
 
-	@bind
-	dispose() {
+	// eslint-disable-next-line local/prefer-class-methods
```

Comment:
It kinda made sense to move back to arrow functions for these cases. Otherwise we'd have to use bind + disable / `this: void`.